inst/qmd_template/make_tag_push_summary.qmd

```diff
@@ -224,20 +226,23 @@ new_station_summary <- station_table(new_matches, "tag")
 reactable(new_station_summary)
 ```
 
-```{r}
+```{r, fig.cap="Figure 7. Number of newly-added detections per receiver. Points represent receiver locations, where colors reflect  the number of detections  and sizes  reflect  the number of fish at that location."}
```

Comment:
You can remove the caption from within the tag (rmd-style) and use the #| stuff below instead (qmd-style)